exchanges/okx/okx_types.go

```diff
@@ -322,67 +417,67 @@ type DeliveryHistoryDetail struct {
-	InstID         string           `json:"instId"`
-	BuyLimit       types.Number     `json:"buyLmt"`
-	SellLimit      types.Number     `json:"sellLmt"`
-	Timestamp      okxUnixMilliTime `json:"ts"`
+	InstrumentType string               `json:"instType"`
+	InstrumentID   string               `json:"instId"`
+	BuyLimit       types.Number         `json:"buyLmt"`
+	SellLimit      types.Number         `json:"sellLmt"`
+	Timestamp      convert.ExchangeTime `json:"ts"`
 }
 
 // OptionMarketDataResponse holds response data for option market data
 type OptionMarketDataResponse struct {
-	InstrumentType string           `json:"instType"`
-	InstrumentID   string           `json:"instId"`
-	Underlying     string           `json:"uly"`
-	Delta          types.Number     `json:"delta"`
-	Gamma          types.Number     `json:"gamma"`
-	Theta          types.Number     `json:"theta"`
-	Vega           types.Number     `json:"vega"`
-	DeltaBS        types.Number     `json:"deltaBS"`
-	GammaBS        types.Number     `json:"gammaBS"`
-	ThetaBS        types.Number     `json:"thetaBS"`
-	VegaBS         types.Number     `json:"vegaBS"`
-	RealVol        string           `json:"realVol"`
-	BidVolatility  string           `json:"bidVol"`
-	AskVolatility  types.Number     `json:"askVol"`
-	MarkVolatility types.Number     `json:"markVol"`
-	Leverage       types.Number     `json:"lever"`
-	ForwardPrice   string           `json:"fwdPx"`
-	Timestamp      okxUnixMilliTime `json:"ts"`
+	InstrumentType string               `json:"instType"`
+	InstrumentID   string               `json:"instId"`
+	Underlying     string               `json:"uly"`
+	Delta          types.Number         `json:"delta"`
+	Gamma          types.Number         `json:"gamma"`
+	Theta          types.Number         `json:"theta"`
+	Vega           types.Number         `json:"vega"`
+	DeltaBS        types.Number         `json:"deltaBS"`
+	GammaBS        types.Number         `json:"gammaBS"`
+	ThetaBS        types.Number         `json:"thetaBS"`
+	VegaBS         types.Number         `json:"vegaBS"`
+	RealVol        string               `json:"realVol"`
+	BidVolatility  types.Number         `json:"bidVol"`
+	AskVolatility  types.Number         `json:"askVol"`
+	MarkVolatility types.Number         `json:"markVol"`
+	Leverage       types.Number         `json:"lever"`
+	ForwardPrice   types.Number         `json:"fwdPx"`
+	Timestamp      convert.ExchangeTime `json:"ts"`
 }
 
 // DeliveryEstimatedPrice holds an estimated delivery or exercise price response.
 type DeliveryEstimatedPrice struct {
-	InstrumentType         string           `json:"instType"`
-	InstrumentID           string           `json:"instId"`
-	EstimatedDeliveryPrice string           `json:"settlePx"`
-	Timestamp              okxUnixMilliTime `json:"ts"`
+	InstrumentType         string               `json:"instType"`
+	InstrumentID           string               `json:"instId"`
+	EstimatedDeliveryPrice string               `json:"settlePx"`
```

Comment:
types.Number I will stop pointing these out but I few more can be converted over?